src/main/java/com/wojtek/parkingmeter/controllers/TicketController.java

```diff
@@ -1,29 +1,86 @@
 package com.wojtek.parkingmeter.controllers;
 
-import com.wojtek.parkingmeter.repositories.TicketRepository;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RestController;
+import com.wojtek.parkingmeter.helpers.*;
```

Comment:
takie importy utrudniają review, bo nie wiadomo skąd się wzięła klasa

Reply:
Jak inaczej powinienem to zrobić?